package.json

```diff
@@ -74,209 +74,209 @@
       "explorer": [
         {
           "id": "iotHubDevices",
-          "name": "Azure IoT Hub"
+          "name": "%azure-iot-toolkit.name%"
```

Comment:
Better to be `azure-iot-toolkit.explorer.name`? :)

Reply:
Okay. I will change it.